crypto/poly1305/poly1305_ieee754.c

```diff
@@ -175,46 +175,46 @@ int poly1305_init(void *ctx, const unsigned char key[16])
          * base 2^32 -> base 2^16
          */
         st->r[1] = (st->r[0] + TWO(52)*TWO(16)*TWO0) -
-                               TWO(52)*TWO(16)*TWO0;
+            TWO(52)*TWO(16)*TWO0;
```

Comment:
I'd argue in favour of preserving the original formatting here. Because it's easier for human to recognize the pattern when they are aligned the current way.

Reply:
Somehow I must have overlooked this one, because I should have reverted these changes (also 182,186,...) immediately before committing.